src/main/java/com/neuvector/DockerRunCommandBuilder.java

```diff
@@ -23,17 +23,9 @@ public DockerRunCommandBuilder(String runtime) {
         cmdList.add("--rm");
     }
 
-    DockerRunCommandBuilder withUserAndGroup(String userIdGroupId) {
-        if (userIdGroupId != null && userIdGroupId.length() > 0) {
-            cmdList.add("-u");
-            cmdList.add(userIdGroupId);
-        }
-        return this;
-    }
-
-    DockerRunCommandBuilder withName(String name) {
```

Comment:
Not really related to the task at hand but this added no value and was not needed.